include/importance_sampling.hpp

```diff
@@ -60,22 +60,35 @@ struct DistributeRaysByImportance {
 				   const unsigned raysPerSample,
 				   unsigned *raysDump) const {
 
-	auto threadsVec = alpaka::workdiv::getWorkDiv<alpaka::Grid, alpaka::Blocks>(acc) *  alpaka::workdiv::getWorkDiv<alpaka::Block, alpaka::Threads>(acc);
-	auto nThreads = threadsVec[0];
+	auto threadsVec = 
```

Comment:
should be threadsExtent or something similar that reflects the 2-Dimensional nature of this variable :)